app/src/main/java/com/x8bit/bitwarden/ui/tools/feature/send/addsend/AddSendContent.kt

```diff
@@ -72,6 +72,28 @@ fun AddSendContent(
         modifier = modifier
             .verticalScroll(rememberScrollState()),
     ) {
+        if (isAddMode && !isShared) {
```

Comment:
Just want to confirm that the expected behavior is to have this scroll behind the top bar?

Reply:
It is. This was double checked with design.